app/models/reports/answer_tagging_report.rb

```
@@ -0,0 +1,233 @@
+        end
+
+        user_stats = user_info.map do |user_id, info|
+          taggable_answer_ids = taggable_data.fetch(user_id, []).to_set
+          cnt_taggable        = taggable_answer_ids.size
+
+          # Filter tags to only those on answers that are taggable for this user.
+          # TODO: confirm with prof — if a reviewer submits multiple responses for the
+          # same round, only the latest submitted response should be counted as taggable.
+          # TaggableAnswersReducer may need to deduplicate by keeping only the most
+          # recently submitted response per (reviewer, round).
+          matching_tags  = tagging_stats.fetch(user_id, []).select { |tag| taggable_answer_ids.include?(tag[:answer_id]) }
```

RuboCop (1.86.2)

[convention] Line is too long. [122/120]

(Layout/LineLength)